src/templates/static/legal/regulation.jsx

```diff
@@ -107,47 +107,47 @@ const Regulation = () => {
                         <map name='planetmap' id='planetmap'>
                             <Area
                                 items={[
-                                    { name: 'Sweden',         x: 105.561, y: 13.568 },
```

Comment:
@suthesan-deriv could you restore all these spaces here? They are needed to align the values for better visual representation.